static/css/mine_house_and_expenses.css

```diff
@@ -69,6 +69,51 @@
   right: 0;
 }
 
+.btn-group button {
```

Comment:
I think that the name of the CSS file is not clear, can you change it to mine.css?

Reply:
yes, fixed